examples/30_extended/fetch_runtimes_tutorial.py

```diff
@@ -202,7 +224,7 @@ def print_compare_runtimes(measures):
 )
 run5 = openml.runs.run_model_on_task(model=rs_pipe, task=task)
 
-
+################################################################################
```

Comment:
Could you please add a `#` to render the 2nd paragraph into the text block, too?